pipelines/test/docs.yaml

```diff
@@ -0,0 +1,63 @@
+name: docs
+
+needs:
+  packages:
+    - man-db
+      texinfo
```

Comment:
is a `-` missing here? 🤔

Reply:
It may be worth including `apk-tools` and `grep` in the dependencies as well?

Reply:
Sure, fixed all of these.  (Note that info was installed, even without the hyphen, but I added it anyway)

Reply:
Yeah, I assumed as much.  This seems really odd.  tagging @imjasonh @jonjohnsonjr for 👀